supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -233,6 +234,8 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
 }
 
 object GenomicsFactory {
+  private val config = ConfigFactory.load().getConfig("backend.providers.PAPIv2.config.filesystems.gcs")
```

Comment:
Pass this path in instead of guessing. The backend name is not necessarily constant.

Reply:
any hints for good & usable source for path in this use case?
One option could also be to but these two configs under google -- they are google spesific after all.

Reply:
Yeah you could either:
 - Make the cloudSDK image part of the backend configuration (eg a field on the `CreatePipelineParameters` used throughout the `GenomicsFactory` class); or
- Make the cloudSDK a single static global value (eg in the top-level google stanza like you suggest) and all PAPI backends reference the same cloud SDK value using a global path

Reply:
Did the second one. 

Documentation doesn't give any instructions how to run code in Intellij Idea / Eclipse / etc and autoimporting sbt doesn't work as nicely as maven or gradle --> using field in CreatePipelineParameters would require some better debugging options in ide to be comfortable to commit :)